anomaly_detector/anomaly_detector.py

```diff
@@ -78,11 +79,12 @@ def _load_data(self, time_span, max_entries, fp=None):
         return data, raw
 
     @TRAINING_TIME.time()
-    # @profile
     def train(self, fp=None, node_map=24):
         """Train models for anomaly detection."""
         start = time.time()
         data, _ = self._load_data(self.config.TRAIN_TIME_SPAN, self.config.TRAIN_MAX_ENTRIES, fp)
+        if fp is not None:
```

Comment:
I would also suggest renaming `fp` to `false_positives` as `fp` is usually used as "file pointer" and might be confusing for new contributors

Reply:
Done